Makefile

```diff
@@ -3,14 +3,15 @@
 	watch-transactions watch-balances watch-reconciliations \
 	view-block-benchmarks view-account-benchmarks
 LICENCE_SCRIPT=addlicense -c "Coinbase, Inc." -l "apache" -v
+GO_INSTALL=GO111MODULE=off go get
```

Comment:
We really shouldn't be turning this flag off... this is the go to dep management in Go and is on by default in all the latest supported releases.

https://dev.to/maelvls/why-is-go111module-everywhere-and-everything-about-go-modules-24k#-raw-go111module-endraw-with-go-113

Reply:
Found a solution: `go run X` for all commands. This still loads the dependency in the `go.mod` and allows for running the command.